SR2024/2023-09-23-place-confirmation.md

```diff
@@ -55,9 +33,32 @@ The shipping address we have for you is:
 So that your kit arrives in time for Kickstart please ensure we have the
 necessary details by 1st October.
 
-We'll send out more information about this year's [Discord][discord] server and
-[Kickstart][kickstart] event in the coming weeks. We look forward to seeing you
-then!
+This year **Kickstart is fully virtual**. We strongly encourage teams to meet up
+to watch the presentations, get to know each other and start working together on
+the kit. We will be shipping kits out to teams ahead of the event, so
+_please ensure we have the correct shipping address_.
+
+On the day we will be livestreaming an introduction to the competition, the game
+and the kits. We will also be providing a set of tasks which will familiarise
+competitors with the kit, which the teams can complete afterwards. There will be
+mentors around in Discord throughout the day to support teams working through
+these tasks.
```

Comment:
If we keep the virtual line but wapt the rest of that paragraph with the next one, it flows better.